task_1.py

```diff
@@ -0,0 +1,10 @@
+# TODO решить с помощью list comprehension и распечатать его
+from pprint import pprint
+
+
+def number_systems(value):
+    dicts_ = [dict([('bin', bin(i)), ('dec', i), ('hex', hex(i)), ('oct', oct(i))]) for i in range(value + 1)]
```

Comment:
dicts_ = [{'bin': bin(i), 'dec': i, 'hex': hex(i), 'oct': oct(i)} for i in range(value + 1)]